code/modules/jobs/job_types/science.dm

```diff
@@ -31,13 +31,14 @@ Research Director
 	name = "Research Director"
 
 	id = /obj/item/weapon/card/id/silver
-	belt = /obj/item/device/pda/heads/rd
+	belt = /obj/item/weapon/storage/belt/utility/full

```

Comment:
yep, {} after a typepath is called a modified type, lets you arbitrarily alter any var on the object, and it'll spawn with those.

(this case doesn't work because my ```user_vars_to_edit``` var is on ```/clothing```, but other memery would work)